docs/website/pages/index.vue

```diff
@@ -71,111 +74,111 @@
     </div>
 
-    <div class="w-3/4 h-auto mx-auto mb-20">
-      <div class="flex flex-col justify-center">
-        <div class="text-center">
-          <h1>Features</h1>
-        </div>
-        <div class="flex justify-center">
-          <div class="text-center w-full px-2 py-2 m-2">
-            <h2>Minimal</h2>
-            <div>
-              Talos is a minimalistic distribution that consists of only a
-              handful of binaries and shared libraries. Just enough to run
-              containerd and a small set of system services. This aligns with
-              NIST's recommendation in the
-              <a
-                href="https://www.nist.gov/publications/application-container-security-guide"
-                target="_blank"
-              >
-                Application Container Security Guide </a
-              >.
-            </div>
-          </div>
-          <div class="text-center w-full px-2 py-2 m-2">
-            <h2>Hardened</h2>
-            <div>
-              There are a number of ways that Talos provides added hardening
-              <ul>
-                <li>
-                  employs the recommended configuration and runtime settings
-                  outlined in the
-                  <a
-                    href="https://kernsec.org/wiki/index.php/Kernel_Self_Protection_Project"
-                    target="_blank"
-                    >Kernel Self Protection Project</a
-                  >
-                </li>
-                <li>
-                  enables mutual TLS for the API
-                </li>
-                <li>
-                  enforces the settings and configurations described in the
-                  <a
-                    href="https://www.cisecurity.org/benchmark/kubernetes/"
-                    target="_blank"
-                    >CIS</a
-                  >
+    <div class="w-3/4 h-auto mx-auto pb-8 mb-20">
+      <div class="text-center">
+        <h1>Features</h1>
+      </div>
+      <div class="md:grid grid-columns-3 grid-gap-12 justify-center">
+        <div class="w-full">
+          <h2 class="text-center">Minimal</h2>
+          <p>
+            Talos consists of only a handful of binaries and shared libraries:
+            just enough to run containerd and a small set of system services.
+          </p>
 
-                  guidelines
-                </li>
-              </ul>
-            </div>
+          <p>
+            This aligns with NIST's recommendation in the
+            <a
+              href="https://www.nist.gov/publications/application-container-security-guide"
+              target="_blank"
+            >
+              Application Container Security Guide</a
+            >.
+          </p>
+        </div>
+        <div>
+          <h2 class="text-center">Hardened</h2>
+          <div>
+            <p>Talos is hardened by design and configuration:</p>
+            <ul class="ml-8">
+              <li>
+                We follow the
```

Comment:
nit: I don't think "We" sounds all that great, but this all is still a WIP so not gonna block. But in looking at a number of other websites they present the "features" in a way that I really don't know how to explain because I'm not an English major :) ... but they put start the sentence with a focus on the point being made instead of "We." Not sure I'm articulating well enough, but I don't like "we" too much.

Reply:
Yeah, I agree with you. I can spend some more time on it and see if I can make more improvements.

Reply:
Let's get in what you have 👍. And we can iterate.